lib/sidekiq_publisher.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+# FIXES: uninitialized constant ActiveSupport::LoggerThreadSafeLevel::Logger
+require "logger"
+require "activerecord"
```

Comment:
This fixes the following `NameError` which I hit in another gem the other day and this was the fix.  I am also explicitly requiring `activerecord` here since have an explicit dependence through `ActiveRecord::Base` now rather than an implicit one through `activerecord-pub-sub`.

```
NameError:
  uninitialized constant ActiveSupport::LoggerThreadSafeLevel::Logger
```